packages/destination-actions/src/destinations/launchdarkly-audiences/syncAudience/index.ts

```diff
@@ -0,0 +1,74 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+import { processPayload } from '../../launchdarkly-audiences/syncAudience/custom-audience-operations'
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Sync Audience',
+  description: 'Sync Engage Audiences to LaunchDarkly segments',
+  defaultSubscription: 'type = "identify" or type = "track"',
+  fields: {
+    custom_audience_name: {
+      label: 'Custom Audience Name',
+      description: 'Name of custom audience list to which user identifier should added/removed',
+      type: 'string',
+      required: true,
+      default: {
+        '@path': '$.context.personas.computation_key'
+      }
+    },
+    segment_computation_action: {
+      label: 'Segment Computation Action',
+      description: "Segment computation class used to determine if action is an 'Engage-Audience'",
+      type: 'string',
+      required: true,
+      default: {
+        '@path': '$.context.personas.computation_class'
+      }
+    },
+    context_kind: {
+      label: 'Context kind',
+      description: 'LaunchDarkly context kind',
+      type: 'string',
+      required: true,
+      default: 'user',
+      choices: [{ label: 'User', value: 'user' }]
```

Comment:
I don't think we want to use choices here because a customer may want to provide a custom value, like `member`
```suggestion
```

Reply:
we might actually be able to remove this field. We dont really send this info to LD because big segments defaults to user context kind. I only added it in user in case users were trying to figure which context is being targeted but as im writing this im realizing it fits better in the docs instead.

Reply:
Good call. I think we may want to add support for specifying the context kind in the capability. However, we can leave it off for now.